src/reason_pprint_ast.ml

```diff
@@ -2152,7 +2152,7 @@ class printer  ()= object(self:'self)
         ]
       (* EQUAL private_flag BAR constructor_declarations {(Ptype_variant _, $2, None)} *)
       | (Ptype_variant lst, scope, None) ->  [
-          privatize scope [makeList ~break:IfNeed ~postSpace:true ~inline:(true, true) (List.map self#type_variant_leaf lst)]
+          privatize scope [makeList ~break:Always_rec ~postSpace:true ~inline:(true, true) (List.map self#type_variant_leaf lst)]
```

Comment:
This seems to do the right thing, but I'm not sure if `Always` should be used here instead?

If I use `Always`, it doesn't insert a newline after the `=`, so maybe it's missing a forced newline there and drop the `_rec`?